examples/2016-10-31/api_swagger_cors/swagger.yaml

```diff
@@ -24,7 +24,7 @@ paths:
           Fn::Sub: "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${LambdaFunction.Arn}/invocations"
 
         passthroughBehavior: when_no_match
-        httpMethod: POST
+        httpMethod: POST  # Keep "POST" when the API definition method is not POST. This "httpMethod" is used to call Lambda.
```

Comment:
Thanks for clarifying this. I've definitely been bitten by this as well. 😅

Consider adding a link to the API Gateway documentation in case someone wants to learn more: https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-integration.html